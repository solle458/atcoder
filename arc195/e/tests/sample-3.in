9 6
765689282 93267307 563699854 951829154 801512848 389123318 924504746 596035433
3 8
2 5
5 8
2 9
8 9
5 7
