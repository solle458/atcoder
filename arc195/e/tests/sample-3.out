55973424
496202632
903509579
343265517
550981449
68482696
